backend/dataall/api/Objects/Environment/schema.py

```diff
@@ -42,6 +42,14 @@
 )
 
 
+EnvironmentParameter = gql.ObjectType(
+    name='EnvironmentParameter',
+    fields=[
+        gql.Field(name='key', type=gql.String),
+        gql.Field(name='value', type=gql.String),
+    ]
+)
+
```

Comment:
[Clarification] - Environment Parameter is actually the link of the Environment with the modules